NickvisionMoney.GNOME/Blueprints/transfer_dialog.blp

```diff
@@ -87,12 +87,20 @@ Adw.Window _root {
             title: _("Conversion Needed");
             visible: false;
 
-            Adw.EntryRow _sourceCurrencyRow {
-              activates-default: true;
-            }
+          Adw.ExpanderRow _rowUseCustomRates {
+                  title: _("Use Custom Rates");
+                  subtitle: "Enter the conversion rates manually";
+                  enable-expansion: false;
+                  show-enable-switch: true;
```

Comment:
Seems like indentations are wrong. If you use IDE that doesn't support blp syntax and so doesn't help you with indentations, set syntax to Less, at least this is what I use in Fleet, it works mostly okay. Or CSS, this should work too.

Reply:
Ah yes. I use Rider which doesn't support blp.

Reply:
> I just Rider too and it doesn't seem to have any issues with blp files, since we have a .editorconfig file. Make sure you open the root of the repo in Rider and not just NickvisionMoney.GNOME and NickvisionMoney.Shared individually...

I'm opening the entire project in rider. I probably just pressed tab twice here 🤦 mb mb...lol